### ISSUE TITLE
Shipping rule cost(s) not being applied to any cart or order

### ISSUE THREAD

**Issue by 23d1** [craftcms/commerce#777]
### Description
Shipping rule costs are not being applied to orders since the last update.

The shipping option(s) show up as selectable under the shipping step in the checkout process, but once the customer reaches the final step with order review, and submits payment, the shipping rule cost is not applied/included. 

The shipping handle/name is added to the cart/order, but the rule adjuster/value is not there.
If I change the shipping cost base rate to something other than 0, that is applied, but the "Category Rate Overrides" or "Per Item Rates" are not applied. So that's the main issue.

Not 100% sure when this issue occurred, but I believe it was the latest update (or possibly 2.1.0). 

The last order shipping was applied to was on Feb 26, so it's been a while before we noticed.
We had to turn off global shipping so this could end up costing a bit in loss of sales.

Edit; Looking through similar issues, this might be related to, if not the same, as #759.

### Additional info
- Craft version: 3.1.17.1
- PHP version: 7.3
- Database driver & version: mariaDB/MySQL 5.5.5
- Plugins & versions: Craft Commerce 2.1.0.2

**Comment by 23d1** [comment]
Not sure if this is related, but when I try to add a new shipping method I get this error;

```
Twig Template Loading Error – craft\web\twig\TemplateLoaderException
Unable to find the template “commerce/_layouts/settingscp” in "commerce/shipping/shippingmethods/_edit" at line 1.
```

**Comment by lukeholder** [comment]
Can you send your DB backup to support@craftcms.com and I will take a look. I can't reproduce the issue. I tested category level item costs, weight costs, and percentage costs, across multiple shipping categories and the calculations seem right.

**Comment by 23d1** [comment]
I've sent a DB backup of my local dev version, which has the exact same issues.

**Comment by lukeholder** [comment]
Thanks, this is now fixed in 2.1.1

**Comment by 23d1** [comment]
The update(s) seem to have solved it. I'll close this out. Thanks again!
